tests/tensorflow/nightly/xlnet-squad.libsonnet

```diff
@@ -66,7 +65,7 @@ local tpus = import "templates/tpus.libsonnet";
   local v2_8 = {
```

Comment:
Should you remove this mixin if there are no working v2-8 configs?

Reply:
Oops, yes done. Also ran the gen script since I forgot.